spec/18-namespaces.md

```diff
@@ -135,24 +137,32 @@ namespace NS3\Sub1;
   <i>namespace-function-or-const:</i>
     function
     const
+
+  <i>namespace-use-group-clauses:</i>
+    <i>namespace-use-group-clause</i>
+    <i>namespace-use-group-clauses</i>  ,  <i>namespace-use-group-clause</i>
+
+  <i>namespace-use-group-clause:</i>
+    <i>name</i>  <i>namespace-aliasing-clause<sub>opt</sub></i>
```

Comment:
This can have a `namespace-function-or-const` prefix -- however only if the there is no such prefix before the use itself. So this will have to be split up into definition for group use (prefix before whole use) and mixed group use (prefix before element).